source/crud/tests/unified/countDocuments-comment.yml

```
@@ -0,0 +1,70 @@
+description: "count"
```

```suggestion
description: "countDocuments-comment"
```

Observation: I see we have existing `countDocuments` tests in `tests/v1`, but this is the first unified test for the method.

---

Done